components/sensor/power_ina219/power_ina219.go

```diff
@@ -221,5 +221,5 @@ func (d *ina219) Readings(ctx context.Context, extra map[string]interface{}) (ma
 		"volts": pm.Voltage,
 		"amps":  pm.Current,
 		"watts": pm.Power,
-	}, handle.Close()
+	}, nil
```

Comment:
For anyone else reading along: the handle gets closed in a `defer` statement up on line 165. How did this ever work in the first place!?

Reply:
This would be implementation dependent.

Since RDK is mostly used on Pis, and Pis have their own special i2c impl, if the Pi i2c impl can be double-closed without issue then this won't cause a problem.

The genericlinux i2c impl cannot be double-closed, so exposes this issue, but there are not so many users of both genericlinux and the ina219 at present

Reply:
Without looking at the pi i2c impl, I would bet that it can be closed as many times as one likes without panicking.